packages/flutter_test/lib/src/matchers.dart

```diff
@@ -663,12 +663,16 @@ Matcher matchesSemantics({
 ///   * [androidTapTargetGuideline], for Android minimum tappable area guidelines.
 ///   * [iOSTapTargetGuideline], for iOS minimum tappable area guidelines.
 ///   * [textContrastGuideline], for WCAG minimum text contrast guidelines.
+///   * [labeledTapTargetGuideline], for enforcing labels on tappable areas.
 AsyncMatcher meetsGuideline(AccessibilityGuideline guideline) {
   return _MatchesAccessibilityGuideline(guideline);
 }
 
 /// The inverse matcher of [meetsGuideline].
 ///
+/// This matcher is used for verifying accessibility guideline does flag the
+/// violations.
```

Comment:
not sure what this sentence is supposed to say...

Reply:
it probably doesn't worth mention it. I will remove it